### PR TITLE
Removed "paste as plain text" 

### LINKED ISSUE #10377: Remove "paste as plain text"
Thanks to @viciousAegis in #10365, "paste" work as "paste as plain text". Which mean we have two different context menu entries that have the same behavior I fear.

It would be nice to remove one of them. I'd suggest removing "paste as plain text". When there is a single feature, adding details is useless.

If eventually, "formatted text" can be transformed into an "paste as html", then it will be useful to have both of them. But since we are not even sure yet we want this feature, removing one of the two paste entry seems a no-brainer

_Originally posted by @Arthur-Milchior in https://github.com/ankidroid/Anki-Android/issues/10365#issuecomment-1047730112_

### PR DESCRIPTION
**Pull request by shivam-2701**
## Purpose / Description
This commit fixes the issue #10377  and removes "paste as plain text" option from the context menu in NoteEditor activity

## Fixes
Fixes #10377 

Fix

## Approach
The issue was solved by hiding the "paste as plain text" menuItem  in 
onPrepareActionMode() method in 

>NoteEditor.kt

Removing the item was causing problems in testing, so I opted to hide it instead with setVisible method.
## How Has This Been Tested?

I has been test on physical device running android 11 and pixel 4 emulator.
It has also been tested using automated test mentioned on the Development Guide of AnkiDroid

> ./gradlew jacocoTestReport

## Learning (optional, can help others)
The resources used are from the issue page which directs to the android documetation

> [developer.android.com/reference/android/widget/TextView#setCustomInsertionActionModeCallback(android.view.ActionMode.Callback)](https://developer.android.com/reference/android/widget/TextView#setCustomInsertionActionModeCallback(android.view.ActionMode.Callback))

> [developer.android.com/reference/android/widget/TextView#setCustomSelectionActionModeCallback(android.view.ActionMode.Callback)](https://developer.android.com/reference/android/widget/TextView#setCustomSelectionActionModeCallback(android.view.ActionMode.Callback))

_Links to blog posts, patterns, libraries or addons used to solve this problem_

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [x] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/anki/NoteEditor.kt [hunk 1]
```diff
@@ -2056,9 +2057,18 @@ class NoteEditor : AnkiActivity(), DeckSelectionListener, SubtitleListener, Tags
             if (menu.findItem(mClozeMenuId) != null) {
                 return false
             }
+            // Test commit
```
Comment (mikehardy) [inline AnkiDroid/src/main/java/com/ichi2/anki/NoteEditor.kt:2060]: No reason a real commit should include a comment like this

```suggestion
```

File: AnkiDroid/src/main/java/com/ichi2/anki/NoteEditor.kt [hunk 2]
```diff
@@ -2056,9 +2057,18 @@ class NoteEditor : AnkiActivity(), DeckSelectionListener, SubtitleListener, Tags
...
             if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N && menu.findItem(mSetLanguageId) != null) {
                 return false
             }
+
+            // Removes paste as plain text from ContextMenu in NoteEditor
+
```
Comment (mikehardy) [inline AnkiDroid/src/main/java/com/ichi2/anki/NoteEditor.kt:2066]: unnecessary newline I think?

```suggestion
```

File: AnkiDroid/src/main/java/com/ichi2/anki/NoteEditor.kt [hunk 3]
```diff
@@ -2069,6 +2079,7 @@ class NoteEditor : AnkiActivity(), DeckSelectionListener, SubtitleListener, Tags
                     getString(R.string.multimedia_editor_popup_cloze)
                 )
             }
+
```
Comment (mikehardy) [inline AnkiDroid/src/main/java/com/ichi2/anki/NoteEditor.kt:2082]: don't alter spacing unnecessarily

```suggestion
```

**Review by mikehardy — CHANGES_REQUESTED** [review]
I attempted to resolve the conflict, and I reviewed it, just nitpicks, it seems like this could go in

**Review by mikehardy — APPROVED** [review]
If it passes CI this could go in I think

**Review by david-allison — APPROVED** [review]

**Comment by david-allison** [comment]
Merged, congrats!!!

If you're on [our Discord](https://discord.gg/qjzcRTx), send me a PM and I'll add some roles

**Pull request merged**